server/src/utils/socketIoUtils.ts

```diff
@@ -83,27 +89,24 @@ export const emitRoadmapEvent = async <T extends ClientEvents>(
 ) => {
   const {
     roadmapId,
-    dontEmitToUserId,
     requirePermission,
     event,
+    dontEmitToUserId,
+    onlyEmitToUserId,
     eventParams: parameters,
   } = { ...emitRoadmapEventParams };
 
-  console.log('Emitting roadmap socket event:');
-  console.log(JSON.stringify(emitRoadmapEventParams, undefined, 2));
-
   const roomName = `roadmap-${roadmapId}`;
   const roadmapSockets = await io.in(roomName).fetchSockets<ISocketData>();
   const socketUserIds = roadmapSockets.map((s) => s.data.user.id);
-  const socketUsers = await User.query()
+  let socketUsers = await User.query()
```

Comment:
```suggestion
  const socketUsers = await User.query()
```
then this could be const